### DISCUSSION TITLE
I2CDriver - Digital Poti DS3502

### DISCUSSION THREAD

**Discussion by nuffman** [arendst/Tasmota#15679] [Q&A] [answered]
Hello,
i have a question regarding the support of the DS3502. Maybe somebody can help me.

With the latest Tasmota Version 11.1.0 also with Tasmota Sensors Version 11.1.0
I2CScan founds the DS3502 at (0x28)
With I2cDriver the Driver 67 isn´t available. Result = {"I2CDriver":"7,8,9,10,11,12,13,14,15,17,18,20,24,29, 31, 36, 41, 42,44,46,48"} 
"I2CDriver67 1" hasn´t an effect.
the command "Wiper" => Result = Unknown

Do i something wrong or do I have to change something?

Thanks for your help.

**Comment by sfromis** [comment, marked as answer]
DS3502 is an optional feature, not included in precompiled binaries. You need to [build](https://tasmota.github.io/docs/Compile-your-build/) a custom binary (not hard to do) including `#define USE_DS3502`.

**Reply by nuffman** [reply]
Thank you very much. it works now.

**Comment by mokusnz** [comment]
This interests me. How can I create my own binary and where? I am a beginner.

**Reply by sfromis** [reply]
https://tasmota.github.io/docs/Compile-your-build/

**Reply by mokusnz** [reply]
I already found it ...
